SU-HSDO/suhumsci
# FourKitchens Release: Sprint 12

# READY FOR REVIEW

## Summary
- No ticket: Fixes a missing Bourbon Library stylesheet error when compiling the theme
- SHS-119: Fixes color pairings for Mega menu levels 
- SHS-98: Update the caption/credit media template 
- SHS-97: The Photo Album works with hb-raised cards
- SHS-122: Update color pairings for secondary accordion utility class
- SHS-49: Ability for site owners to add hb-raised-cards and hb-raised-cards--uniform-height on Collections

## Need Review By (Date)
- 11/18

## Urgency
low

## Steps to Test
1. Testing missing Bourbon library stylesheet
    - Go to `humsci_basic` and run `npm run build:sass` - verify the stylesheets compile and it no longer complains about a missing stylesheet
2. Testing the mega menu link colors
    - Go to `/admin/config/site-options` and `admin/appearance/settings/humsci_colorful` to enable the mega menu
    - On `admin/appearance/settings/humsci_colorful` (or traditional) adjust the color pairings and `Local Masthead Variant` to various options. Verify the following adjustment occur:
    - The top level links remain the global color of white (default masthead setting) and black (dark masthead setting)
    - The hover color bad is the `secondary` color
    - The second level is `palette--tertiary` with the hover state being `palette--tertiary-darken-20`
    - The third level is global black
3. Testing Caption/Credit media templates
    - Go to hs-colorful.suhumsci.loc/components/photo-album and verify the captions still display and are unchanged when compared to production
    - Go to hs-traditional.suhumsci.loc/components/photo-album and verify the captions still display and are unchanged when compared to production
5. Testing `.hb-raised` class for Photo Albums
    - Create a flexible page. Add a collection. Verify there is the 'Raised Cards' checkbox option 
    - Check the raised cards box and in the collection add a View of Default Events (cards), a vertical timeline and a testimonial.
    - Add another collection with a couple options from above but do not check the raised cards box.
    - View the page and verify the raised cards collection has the effect and the one without the checked box does not.
    - View the preexisting mixed raised cards page and make sure the work did not adversely affect this page (/mixed-page-wells-raised-cards)
6. Testing new color pairings for secondary accordion styles
    - Go to the accordions page (/accordions). Edit the layout, under the Main Region, add the hb-accordion-secondary class to the Field Formatter Class area.
    - Visit the accordion page and verify that it's using the correct layout (border color of the theme's secondary color, hover is the background primary color etc)
    - Go to the theme settings (/admin/appearance/settings/humsci_colorful) under Color Pairing change the color to a different color. Go back to the accordions page and verify it changed to the appropriate color. Do this a few times to be sure.
    - Repeat these steps for the Traditional site.


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
